joomla/joomla-cms
# [4.0]  Enabling stemming in com_finder for mono-lingual sites

This PR extends the work of #20391 and while it can be tested the way it is, #20391 should be merged first and later this here.

This PR adds a new option to the configuration of Finder/Smart Search to select the default stemmer to use for content items marked as "All" or monolingual sites. You can select "None", a specific language or the default language for the frontend, that is set in the language manager.

All the code for this feature is in FinderIndexerHelper::stem().

## How to test
1. Install 4.0 with sample data
2. Run indexer in the backend and see that content set to "All" in the DB in #__finder_terms is not stemmed.
3. Go to component configuration and set the default stemmer to your prefered setting.
4. Clear Index and start Index again, noticing that content set to "All" is now stemmed, too.
